components/lightsnapcast/player.c

```diff
@@ -625,25 +631,36 @@ int32_t latency_buffer_full(bool *is_full, TickType_t wait) {
 /**
  *
  */
-int32_t get_diff_to_server(int64_t *tDiff) {
-  static int64_t lastDiff = 0;
+int32_t get_diff_to_server(int64_t *tDiff, int64_t now) {
+  static double lastDiff = 0;
```

Comment:
Won't you need to reset those static variables at any time?

Reply:
Not sure if I understand your question?